### PR TITLE
Add CVE-2022-24825 for GHSA-gcj7-j438-hjj2

### PR DESCRIPTION
**Pull request by CallmeMari**
Add CVE-2022-24825 for GHSA-gcj7-j438-hjj2

### CODE REVIEW

File: 2022/24xxx/CVE-2022-24825.json [hunk 1]
```diff
@@ -1,18 +1,88 @@
 {
```
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24825.json:1]: 'github.com/stripe/smokescreen' does not match '^(ftp|http)s?://\\S+$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/24xxx/CVE-2022-24825.json [hunk 2]
```diff
@@ -1,18 +1,88 @@
...
+    },
+    "references": {
+        "reference_data": [
+            {
+                "name": "https://github.com/stripe/smokescreen/security/advisories/GHSA-gcj7-j438-hjj2",
+                "refsource": "CONFIRM",
+                "url": "https://github.com/stripe/smokescreen/security/advisories/GHSA-gcj7-j438-hjj2"
+            },
+            {
+                "name": "github.com/stripe/smokescreen",
```
Comment (anthnysingleton) [inline 2022/24xxx/CVE-2022-24825.json]: Hey @CallmeMari,

Looks like the "name" and "url" attributes both are missing  "https://". Once that is fixed the process will continue and merge the PR. 

Thanks,
CVE Content Team

Comment (CallmeMari) [reply 2022/24xxx/CVE-2022-24825.json]: Thank you for catching this, it has been fixed.

**Pull request merged**